content/docs/1.8.0/advanced-resources/backing-image/backing-image.md

```diff
@@ -154,6 +159,27 @@ Since v1.3.0, users can download existing backing image files to the local via U
 #### Notice:
 - Users need to make sure the backing image existence when they use UI to create or restore a volume with a backing image specified.
 - Before downloading an existing backing image file to the local, users need to guarantee there is a ready file for it.
+- Downloading a v2 data engine backing image is currently not supported.
+
+## Create V2 Data Engine Backing Image
+Since v1.8.0, users can create a backing image supported by v2 data engine by specifying `Data Engine` in the YAML, through the UI or StorageClass.
+
+### Parameters during creation
+All the parameters are the same as the v1 data engine backing image except `Data Engine`.
+
+#### The data source of a backing image
+You can prepare a v2 data engine backing image using five different kinds of data sources as v1.
+1. Download a backing image file (using a URL).
+2. Upload a file from your local machine. This option is available to Longhorn UI users.
+3. Export an existing in-cluster v1 data engine volume as a backing image.
+4. Restore a backing image from the backupstore, For more information, see [Backing Image Backup](../backing-image-backup).
+5. Clone a v1 backing image.
+
+#### Notice:
+- Exporting from a v2 data engine volume is currently not supported
+- Cloning a v2 backing image is currently not supported.
+- Backing up a v2 backing image is currently not supported.
+- For qcow2 image, Longhorn first converts it into raw image and stores the data into v2 data engine backing image due to the v2 data engine limitation.
```

Comment:
```suggestion
#### Notice

- The following operations are currently not supported:
  - Exporting from a V2 Data Engine volume
  - Cloning a V2 backing image
  - Backing up a V2 backing image
- Because of a V2 Data Engine limitation, Longhorn first converts a qcow2 image to a raw image, and then stores the data in a V2 Data Engine backing image.
```

Reply:
fixed.